physics/CONV/Chikira_Sugiyama/cs_conv.F90

```diff
@@ -492,9 +500,9 @@ subroutine cs_conv_run(         IJSDIM ,  KMAX     , ntracp1 , NN,       &
 !          CNV_PRC3(i,k)  = 0.0
            CNV_NDROP(i,k) = 0.0
            CNV_NICE(i,k)  = 0.0
-           cf_upi(i,k)    = max(0.0,min(0.01*log(1.0+500*ud_mf(i,k)),0.25))
+!          cf_upi(i,k)    = max(0.0,min(0.01*log(1.0+500*ud_mf(i,k)),0.1))
 !    &                                               500*ud_mf(i,k)),0.60))
-           CLCN(i,k)      = cf_upi(i,k)                     !downdraft is below updraft
+!          CLCN(i,k)      = cf_upi(i,k)                     !downdraft is below updraft
            
            w_upi(i,k)     = ud_mf(i,k)*(t(i,k)+epsvt*gdq(i,k,1)) * rair &
```

Comment:
@AnningCheng-NOAA I'm guessing that this doesn't get executed since `do_aw` is true when this scheme is executed in UFS, but if this branch of code does get executed, the calculation of `w_upi` is referencing `cf_upi` that no longer is given a value.

Reply:
I prefer to give a value to cf_up by uncommenting line 495.